### PR TITLE
Preserve site without api

### PR DESCRIPTION
**Pull request by Isti01**

### CODE REVIEW

File: src/models/static/breaksData.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+export const breaksData = [
```
Comment (berenteb) [inline src/models/static/breaksData.ts]: Ide egy explicit típus jól jöhet, mert kimutatja ha véletlen beleírok valamit rosszul

**Review by berenteb — CHANGES_REQUESTED** [review]

File: src/models/static/breaksData.ts (hunk 1 above)
Comment (Isti01) [reply src/models/static/breaksData.ts]: Done

**Review by berenteb — APPROVED** [review]

File: src/app/actions.ts [hunk 2]
```diff
@@ -35,45 +35,6 @@ export async function addToGroup({ email }: { email: string }) {
...
-async function validateRecaptcha(token: string) {
-  const res = await fetch('https://www.google.com/recaptcha/api/siteverify', {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/x-www-form-urlencoded',
-    },
-    body: `secret=${process.env.RECAPTCHA_SECRET}&response=${token}`,
-  });
-  const data = await res.json();
-  return data.success;
```
Comment (Tschonti) [inline src/app/actions.ts:78]: It would've been better if this was left here as a comment, could be a big help next year.
Also at the network functions

**Review by Tschonti — COMMENTED** [review]
Thanks!

**Pull request merged**